packages/components/src/form-toggle/style.scss

```diff
@@ -1,13 +1,18 @@
 $toggle-width: 36px;
 $toggle-height: 18px;
 $toggle-border-width: 1px;
+$thumb-size:  $toggle-height - ($toggle-border-width * 6);
+
+$transition-duration-color: 0.2s;
+$transition-duration-position: 0.1s;
```

Comment:
Any reason why these shouldn't be the same 0.2 value?

Reply:
Here it is with 0.2s on both: 

https://github.com/WordPress/gutenberg/assets/1813435/27b69cf3-33e7-4a70-972d-7f1fcc1fa1a0

Reply:
No reason, I kept the same transition durations as currently on `trunk`. Happy to have all durations be the same — `0.2s` or any other value that you would recommend

Reply:
I went ahead and updated all transitions to have the same duration (`0.2s`). I'll go ahead and merge this PR, but I'm happy to iterate further in a follow-up if we need to make further tweaks